3dhubs_amsterdam_python_flask.html

```diff
@@ -0,0 +1,64 @@
+---
+title:  Python Developer
+company: 3D Hubs
+url: https://www.3dhubs.com/
+location: Amsterdam, the Netherlands
+contract: temporary but can be converted to permanent 
+contact:
+    name: Mrs A. Cojanu
+    email: alina@3dhubs.com
+    phone: 0031 6 81 11 41 17
+created: !!timestamp '2017-01-27' 
+tags:
+  - the Netherlands
+  - Amsterdam
+  - python
+  - postgresql
+  - flask
+  - medior 
+  - senior
+---
+
+# Overview
+
+At 3D Hubs we are doing something that has never been done before; decentralising manufacturing by connecting people who want to 3D print to the people owning the machines.
+Today, 3D Hubs provides over 1 billion people access to a 3D printer within 10 miles of their home, that's almost 15% of the world population.
+
+As our new Python Engineer you are committed to clean, stable and scalable code, that creates and connects content to the API.
+You have done large scale projects in Python and Flask, SQLAlchemy or similar. 
+Databases have no secrets to you and you can speak the language of complex queries. 
+You are wired like an entrepreneur, quickly grasping the insights and taking action.
+Your sidekicks will be colleagues from these awesome squads - Geometry, Demand, Supply, Evolution, Supply and Ops.
+Together we will build scalable solutions and literally change the way manufacturing will work in the future. 
+
+Are you up for this challenge? Read on!
+
+# Our technology stack
+
+- PHP (Drupal 7/Composer)
+- Python3 (Flask/SQLAlchemy)
+- PostgreSQL
+- RabbitMQ
+- ES6 (Angular 1.5, Three.js)
+- Sass / Pug
+- Docker
+- AWS: EC2, S3, RDS etc.
+
+# You
+
+- Get excited by shipping your solutions and getting them validated by users asap
+- Are eager to teach your colleagues and share knowledge inside and outside the team
+- Like to choose the right tool for the right job
+- Have experience in test automation
+- Enjoy a vibrant, creative and ever changing environment
+
+# Perks
+
+- Free 3D printing, anytime, at our office
+- Real impact on our technical roadmap
+- Work in a lean, fast-paced, start-up environment located in the heart of Amsterdam - Frederiksplein  
+- Work together with a rapidly expanding, international, young team of 40+ people and 12+ nationalities
+- Get time to work on your own projects at our Pizza Nights
+- Free lunch, Italian quality coffee and awesome Friday drinks
+
+If interested please follow this link [jobs.lever.co/3dhubs](https://jobs.lever.co/3dhubs/2273b01a-fb11-49a7-8b1f-a745a7357d63?lever-origin=applied&lever-source%5B%5D=githubpython)
```

Comment:
You should use this URL at the top in the metadata - it's the URL you want interested candidates to click.